packages/gatsby/src/redux/actions/public.js

```diff
@@ -480,10 +480,11 @@ ${reservedFields.map(f => `  * "${f}"`).join(`\n`)}
 
   let deleteActions
   let updateNodeAction
+  // marking internal-data-bridge as owner of SitePage instead of plugin that calls createPage
```

Comment:
Before changes in this PR we just never actually validate type owners of `SitePage` node types, as validation was happening in createNode action creator - moving validation to reducer exposed it, so this just make sure there is uniform type owner of SitePage. `internal-data-bridge` is plugin that creates nodes for various of primitives in gatsby - such as `Site` or `SiteFunction`, it used to create `SitePage` as well in the past but it was moved here for performance reason

Reply:
ah that makes sense, thank you!